metron-platform/metron-parsers/src/main/java/org/apache/metron/parsers/bolt/ParserBolt.java

```diff
@@ -117,7 +117,7 @@ public void execute(Tuple tuple) {
       int numWritten = 0;
       if(sensorParserConfig != null) {
         List<FieldValidator> fieldValidations = getConfigurations().getFieldValidations();
-        Optional<List<JSONObject>> messages = parser.parseOptional(originalMessage);
+        Optional<List<JSONObject>> messages = parser.parseOptional(originalMessage, sensorParserConfig);
```

Comment:
This patch is premised on the idea that changes to the sensorParserConfig will be recognized and passed in to the GrokParser from the ParserBolt.  Sorry I'm not familiar enough with the code, but I can't find where the sensorParserConfig is refreshed, or how often it happens.  Can you please point it out?  Thanks.